src/elements/Label/Label.js

```diff
@@ -35,31 +34,34 @@ function Label(props) {
     useKeyOnly(circular, 'circular'),
     useKeyOnly(floating, 'floating'),
     useKeyOnly(horizontal, 'horizontal'),
+    useKeyOnly(image, 'image'),
     useKeyOnly(empty, 'empty'),
     useKeyOnly(tag, 'tag'),
     useValueAndKey(attached, 'attached'),
     useKeyOrValueAndKey(corner, 'corner'),
     useKeyOrValueAndKey(pointing, 'pointing'),
+    /* Don't change order of `above` and `below`, they must be after `pointing` */
+    useKeyOnly(above, 'above'),
+    useKeyOnly(below, 'below'),
```

Comment:
I think since this is a special case that is not reused, we should not create utilities for it.  We should just handle it directly in the component:

```jsx
Label._meta = {
  props: {
    pointing: ['left', 'right', 'above', 'below']
  }
}

function Label() {
  const pointingClass = pointing && 'pointing'
    || _.includes(['left', 'right'], pointing) && `${pointing} pointing`
    || _.includes(['above', 'below'], pointing) && `pointing ${pointing}`

  cx(
    pointingClass
  )
}
```